packages/m2m-gateway/src/services/purposeService.ts

```diff
@@ -37,9 +37,9 @@ export const sortPurposeVersionsByDate = (
 export const getPurposeCurrentVersion = (
   purpose: purposeApi.Purpose
 ): purposeApi.PurposeVersion | undefined => {
-  const statesToExclude: m2mGatewayApi.PurposeVersionState[] = [
-    m2mGatewayApi.PurposeVersionState.Values.WAITING_FOR_APPROVAL,
-    m2mGatewayApi.PurposeVersionState.Values.REJECTED,
+  const statesToExclude: purposeApi.PurposeVersionState[] = [
+    purposeApi.PurposeVersionState.Values.WAITING_FOR_APPROVAL,
+    purposeApi.PurposeVersionState.Values.REJECTED,
```

Comment:
Unrelated small fix based on @taglioni-r's [comment](https://github.com/pagopa/interop-be-monorepo/pull/1918#discussion_r2104685619)